samples/snippets/fsharp/VS_Snippets_CLR/Decimal Example/FS/fs.fsproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you move all the snippet files up a level? We removed the top-level "samples" folder recently.

Reply:
👍